### DISCUSSION TITLE
[A-L] (2023/24) Lezione 09/10/2023 - Esercizio 2

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#173] [Esercizi - Viaggi/Piazza]
**Esercizio 2:** Sia $(G,\star)$ un gruppo, verificare che $(g\ \star\ h)^{−1}= h^{−1}\ \star\ g^{−1}, \forall g, h \in G$.
> Suggerimento: utilizzate l’unicità dell’inverso.

**Comment by Elia-Belli** [comment]
Dimostriamo che entrambi i lati dell'equazione sono inversi per $(g \star h)$:
- Per definizione di elemento inverso: $(g \star h)^{-1} \star (g \star h) = e$;

- Sia $k = h^{-1} \star g^{-1}$, sfruttando l'associatività di $\star$:
 $(g \star h) \star k= g \star (h \star k)= g \star (h \star (h^{-1} \star g^{-1})) = g \star ((h \star h^{-1}) \star g^{-1})= g \star (e \star g^{-1}) = g \star g^{-1} = e$

Allora $(g \star h)^{-1}$ e $(h^{-1} \star g^{-1})$ sono entrambi inversi per $(g \star h)$, siccome l'inverso è unico allora essi sono uguali.

**Comment by Elia-Belli** [comment]
@CarloDaRomadev ci sono un paio di errori nella tua dimostrazione:
- hai supposto che il gruppo fosse commutativo, ma $G$ si tratta di un gruppo generico, quindi non puoi scambiare l'ordine degli elementi (puoi solamente decidere se farla prima con l'elemento sinistro o destro, per l'associatività)
- $(g \star h)^{-1}$ non è inverso per $(h \star g)$ sempre perchè non vale la commutatività
-  non hai utilizzato l'unicità dell'inverso :(

**Comment by daromacarlo** [comment]
@Elia-Belli ma porca pupazza, hai perfettamente ragione

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/adc21083-df08-428e-82e5-ad67ad9732e0)
